src/util/ui.js

```diff
@@ -1,3 +1,10 @@
+const smallNumFormatter = new Intl.NumberFormat('en-US');
+
+const largeNumFormatter = new Intl.NumberFormat('en-GB', {
```

Comment:
Should be `'en-US'`, right?
```suggestion
const smallNumFormatter = new Intl.NumberFormat('en-US');

const largeNumFormatter = new Intl.NumberFormat('en-US', {
```